nvm.sh

```diff
@@ -369,6 +373,8 @@ nvm_version_dir() {
     nvm_echo "${NVM_DIR}/versions/node"
   elif [ "_${NVM_WHICH_DIR}" = "_iojs" ]; then
     nvm_echo "${NVM_DIR}/versions/io.js"
+  elif [ "_${NVM_WHICH_DIR}" = "_chakra" ]; then
```

Comment:
chakra isn't the same category as "node" or "io.js" - it's an engine, not a platform. node and io.js are implicitly v8, so I'd want an entirely new way to explicitly specify the *engine* that you're using with the particular node version (and alias "v8" to the current implicit default)

The engine here is more akin to the way I've been planning to support nightlies and release candidates; and I'll want to make sure that engine support both works well with those, and doesn't force a specific implementation path for them.

Reply:
Ok, what syntax do you imagine then?

Reply:
for RCs, I'd initially begun with `iojs:rc-v1.2.3` and `node:rc-v4.5.6` (which would be the same here; you could do `node:chakra`, for example).

However, my in-progress branch also does some significant refactoring of the internals of nvm to be able to achieve this cleanly.